met-api/src/met_api/resources/contact.py

```diff
@@ -88,3 +88,26 @@ def get():
             return ActionResult.success(result=widgets)
         except (KeyError, ValueError) as err:
             return ActionResult.error(str(err))
+
+
+@cors_preflight('PUT')
+@API.route('/')
+class Contact(Resource):
```

Comment:
you don't need to create a new class for this, just use the first one on this file (you can add the 'PUT' allowed method if needed), also I noticed the class names are wrong (submission/survey stuff) rename them to Contact/Contacts

Reply:
thank you for all the changes i addressed them